packages/client/src/render/screenRouter.js

```diff
@@ -43,7 +43,7 @@ export const screenRouter = ({ screens, onScreenSelected, window }) => {
     return sanitize(url)
   }
 
-  const routes = screens.map(s => makeRootedPath(s.route))
+  const routes = screens.map(s => makeRootedPath(s.routing?.route))
```

Comment:
Ah no problem, easy enough to remove this instance of it though, just something to remember I guess for the client!

Might be worth while putting Babel into the client build process purely so that we can use a similar style as the rest of the frontend to avoid issues like this.

Reply:
For what it's worth, I've updated the new client to support nice features like optional chaining in the client rewrite branch! This specific file will be deleted anyway, but going forward we can use these features in the client lib. Don't need babel for it though since svelte is already a compiler - so just needed to update the versions of svelte and a few of the rollup plugins to support it.